Examples/BookingSystem.AspNetCore/Controllers/OpenBookingController.cs

```diff
@@ -198,5 +198,12 @@ public IActionResult TestInterfaceAction([FromServices] IBookingEngine bookingEn
             }
```

Comment:
This file is not copied to Examples/BookingSystem.AspNetFramework in the script btw. What's the process for getting these changes into the AspNetFramework project?

Reply:
The same features just need to be added manually, as the controllers in both projects follow a slightly different convention